pep-0597.rst

```diff
@@ -33,135 +34,140 @@ Using the default encoding is a common mistake
 Developers using macOS or Linux may forget that the default encoding
 is not always UTF-8.
 
-For example, ``long_description = open("README.md").read()`` in
-``setup.py`` is a common mistake. Many Windows users can not install
-the package if there is at least one non-ASCII character (e.g. emoji)
-in the ``README.md`` file which is encoded in UTF-8.
+For example, using ``long_description = open("README.md").read()`` in
+``setup.py`` is a common mistake. Many Windows users cannot install
+such packages if there is at least one non-ASCII character (e.g. emoji)
+in their UTF-8-encoded ``README.md`` file.
 
-For example, 489 packages of the 4000 most downloaded packages from
-PyPI used non-ASCII characters in README. And 82 packages of them
-can not be installed from source package when locale encoding is
-ASCII. [1]_ They used the default encoding to read README or TOML
-file.
+Of the 4000 most downloaded packages from PyPI, 489 use non-ASCII
+characters in their README, and 82 of these do not explicitly specify
+the encoding used to open it. [1]_ Therefore, they cannot be installed
+from source in non-UTF-8 locales.
```

Comment:
```suggestion
characters in their README, and 82 fail to install from source on
non-UTF-8 locales due to not specifying an encoding for a non-ASCII
file. [1]_
```

Per the [discussion](https://github.com/python/peps/pull/1887/files#r600248579) previously, this should be the last needed change; lmk if you want a different proposed version or it to say something else. Once this is LGTM, I'll batch commit everything in and this should be good to merge, thanks.

Reply:
lgtm